lib/developer_portal/app/views/shared/cms/_toolbar.html.slim

```diff
@@ -1,7 +1,6 @@
 / Order matters! Base must go before any other pack:
 = stylesheet_packs_chunks_tag 'patternfly_base'
 = javascript_packs_with_chunks_tag 'dev_portal_cms_toolbar'
-= stylesheet_link_tag "font-awesome" unless Rails.env.test?
```

Comment:
We'll just use FontAwesome v5 from Patternfly 4 (included from `patternfly_base`).